### PR TITLE
santiago-aldanondo

### PR DESCRIPTION
**Pull request by santiagoaldanondo**

### CODE REVIEW

File: starter-code/database.js [hunk 1]
```diff
@@ -1,145 +1,173 @@
-const MongoDB = require('mongodb');
-const users = 'users';
-const products = 'products';
-const shoppingCarts = 'shoppingCarts';
+const MongoDB = require('mongodb')
 
-class Database {
+// Let us use ObjectId. It is not needed for this exercise, but I keep it to have it for the future
+var ObjectId = require('mongodb').ObjectId
```
Comment (ta-web-mad) [inline starter-code/database.js:4]: Use let/const instead of var

File: starter-code/database.js [hunk 2]
```diff
@@ -1,145 +1,173 @@
...
       } else {
         // LAB 6
         // Implement the query to delete a product
-        // productName is the name of the producto to delete 
+        // productName is the name of the producto to delete
         // remeber once it's finish to comment callback('Error deleting product');
-        
-        callback('Error deleting product');
+
+        console.log(productName)
```
Comment (ta-web-mad) [inline starter-code/database.js:132]: Clean the code before commiting, do not push code with console outputs like this. :)

File: starter-code/main.js [hunk 3]
```diff
@@ -1,234 +1,238 @@
...
-				break;
-		}
-	})
+function mainMenu () {
+  clear()
+  questions.showMainMenu()
+  questions.typeAnOption((option) => {
+    switch (option) {
+      case '1':
+        usersMenu()
```
Comment (ta-web-mad) [inline starter-code/main.js:14]: Semicolons

File: starter-code/main.js [hunk 4]
```diff
@@ -1,234 +1,238 @@
...
+function insertUser () {
+  questions.askingInsertUser((user) => {
+    db.insertUser(user, (error, result) => {
+      if (error) {
+        console.log(error)
+        questions.continue(() => {
+          usersMenu()
+        })
+      } else {
+        console.log('Inserted: ', result.result.n)
```
Comment (ta-web-mad) [inline starter-code/main.js:76]: Oops, console.log...

File: starter-code/main.js (hunk 3 above)
Comment (santiagoaldanondo) [reply starter-code/main.js:14]: Semicolons are automatically removed by Eslint... not my fault this time :)

File: starter-code/database.js (hunk 1 above)
Comment (santiagoaldanondo) [reply starter-code/database.js:4]: Sorryyyyyyy

**Pull request closed without merging**